### DISCUSSION TITLE
Any way to override specific icons for todo items?

### DISCUSSION THREAD

**Discussion by Anrock** [nvim-neorg/neorg#530] [Q&A] [answered]
While I find cross and mark icons fine for pending and completed tasks, a symbol for `?` tasks is quite opaque for me.

Is there a way to override those symbols?

**Comment by max397574** [comment, marked as answer]
it's possible in the concealer config

**Reply by Anrock** [reply]
Is there an example somewhere? I've checked [concealer wiki page ](https://github.com/nvim-neorg/neorg/wiki/Concealer) and there is icon section but default value is empty, so I have no idea of a structure for todo icons. It seems like presets don't touch todo icons too.

**Reply by Anrock** [reply]
I think I've found it
https://github.com/nvim-neorg/neorg/blob/main/lua/neorg/modules/core/norg/concealer/module.lua#L1003
